src/js/components/invoice/AddItemsPage.jsx

```diff
@@ -150,12 +153,13 @@ class AddItemsPage extends Component {
    */
   setInvoiceItems(response, startIndex) {
     this.props.showSpinner();
-    const { data } = response.data;
+    const { data, totalCount } = response.data;
```

Comment:
I'm not sure if this is a good idea to set totalCount each time new lines are loaded. Perhaps we could update totalCount when items in modal are saved?

Reply:
This is evaluated in `onResponse` when items in modal are saved.